### PR TITLE
Announce membership programme

### PR DESCRIPTION
**Pull request by thibaultamartin**

### CODE REVIEW

File: content/blog/2023/06/2023-06-20-membership-program.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+or our [fight against the Online Safety Bill](https://matrix.org/blog/2021/05/19/how-the-u-ks-online-safety-bill-threatens-matrix)).
+
+**A curated protocol benefits profit-making companies building communication
+products and services.** Decentralisation is a hard problem. End-to-end
+encryption is a hard problem. Decentralised end-to-end encrypted communications
+are a very hard problem. Matrix is doing the heavy lifting in these areas so
+companies can focus on building what they do best: creating great user
+experiences on top of it. By financially supporting the project, organisations
+building on Matrix rest assured it keeps evolving, being patched and staying
+secure. In short it ensures Matrix remains a competitive advantage over other
```
Comment (jplatte) [inline content/blog/2023/06/2023-06-20-membership-program.md:59]: "remainds advantaged" ("competitively"?) or, easier:

```suggestion
secure. In short it ensures Matrix maintains a competitive advantage over other
```

another alternative:

```suggestion
secure. In short it ensures Matrix keeps a competitive advantage over other
```

File: content/blog/2023/06/2023-06-20-membership-program.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+the Matrix Foundation, public sector entities contribute to the stability,
+security and performance of Matrix. Investing into the project ensures that the
+open standard that is supporting their healthcare, defence and public
+administration continues to benefit from innovative open source software
+development.
+
+
+## Stay unbiased & vendor neutral
+
+The Matrix manifesto hasn’t changed. We still believe people should have full
```
Comment (jplatte) [inline content/blog/2023/06/2023-06-20-membership-program.md:75]: Should we use regular ASCII apostrophes, or are the ones you used somehow preferable?

File: content/blog/2023/06/2023-06-20-membership-program.md [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+control over their own communication, not be locked in silos, be able to
+converse securely and privately, and that communication should be available to
+everyone as a free and open, unencumbered, standard and global network.
+
+The Foundation maintains the Spec, which formalises the behaviour expected from
```
Comment (jplatte) [inline content/blog/2023/06/2023-06-20-membership-program.md:80]:
```suggestion
The Foundation maintains the Matrix Specification, which formalises the behaviour expected from
```

File: content/blog/2023/06/2023-06-20-membership-program.md [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+the various software components of the Matrix ecosystem. We believe it’s
+important for both individuals using the protocol - and organisations building
+products on top of it - to be part of the decision process when it comes to
```
Comment (jplatte) [inline content/blog/2023/06/2023-06-20-membership-program.md:83]:
```suggestion
products on top of it - to be part of the decision-making process when it comes to
```

Small style nit, "decision-making process" seems like a very common term while I don't think I've seen "decision process" before.

File: content/blog/2023/06/2023-06-20-membership-program.md [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+shaping the evolution and functionalities of Matrix. So we have designed
```
Comment (jplatte) [inline content/blog/2023/06/2023-06-20-membership-program.md:84]: "functionalities" also seems like a very uncommon word, would "features" match the intended meaning?

File: content/blog/2023/06/2023-06-20-membership-program.md [hunk 6]
```diff
@@ -0,0 +1,150 @@
...
+
+## Going where we’re expected to
+
+The Governing Board is the new compass of the Foundation: it refines the vision
+of the Foundation and steers the Managing Director and Spec Core Team in the
+right direction.
+
+This gives the Governing Board a lot of power over the Foundation and Matrix,
+so we’ve put in place several safeguards to ensure the Foundation remains
+healthy.
```
Comment (jplatte) [inline content/blog/2023/06/2023-06-20-membership-program.md:123]: Extra trailing space.

File: content/blog/2023/06/2023-06-20-membership-program.md (hunk 1 above)
Comment (thibaultamartin) [reply content/blog/2023/06/2023-06-20-membership-program.md:59]: I mean to say using Matrix is a competitive advantage, I'm not sure your suggestions keep that meaning

**Review by jplatte — APPROVED** [review]

File: content/blog/2023/06/2023-06-20-membership-program.md (hunk 1 above)
Comment (jplatte) [reply content/blog/2023/06/2023-06-20-membership-program.md:59]: No it doesn't, but it was not clear to me at all that that is what this was trying to say.

**Pull request merged**